src/Microsoft.ML.KMeansClustering/KMeansStatic.cs

```diff
@@ -5,11 +5,10 @@
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.KMeans;
-using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
 using System;
 
-namespace Microsoft.ML.Trainers
+namespace Microsoft.ML.StaticPipe
```

Comment:
>StaticPipe [](start = 23, length = 10)

why change namespace?

Reply:
I made this sweeping change, because otherwise we are seeing 2 methods per trainer in the intellisense by default. 

I think for now we want users to 'opt in' to the static API by using `StaticPipe`, and have the dynamic API available right off the bat

---
In reply to: [224950522](https://github.com/dotnet/machinelearning/pull/1252#discussion_r224950522) [](ancestors = 224950522)